clients/src/test/java/org/apache/kafka/test/Microbenchmarks.java

```diff
@@ -78,34 +78,30 @@ public static void main(String[] args) throws Exception {
         final Time time = Time.SYSTEM;
         final AtomicBoolean done = new AtomicBoolean(false);
         final Object lock = new Object();
```

Comment:
Maybe we should just delete this old class ...

Reply:
I'm not sure whether anyone uses this. I still think we should check before deleting it.